packages/order-utils/src/index.ts

```diff
@@ -68,7 +68,7 @@ export {
 } from '@0x/types';
 
 export {
-    OrderError,
+    TypedDataError as OrderError,
```

Comment:
@fabioberger should I export this as `TypedData` and add it to CHANGELOG?

Reply:
Yes, let's make this change.